src/Tools/ExternalAccess/FSharp/Navigation/IFSharpCrossLanguageSymbolNavigationService.cs

```diff
@@ -0,0 +1,28 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Threading;
+using System.Threading.Tasks;
+using Microsoft.CodeAnalysis.Navigation;
+
+namespace Microsoft.CodeAnalysis.ExternalAccess.FSharp.Navigation
+{
+    /// <inheritdoc cref="ICrossLanguageSymbolNavigationService"/>
+    internal interface IFSharpCrossLanguageSymbolNavigationService
+    {
+        /// <inheritdoc cref="ICrossLanguageSymbolNavigationService.TryGetNavigableLocationAsync"/>
+        Task<IFSharpNavigableLocation?> TryGetNavigableLocationAsync(string documentationCommentId, CancellationToken cancellationToken);
+    }
```

Comment:
@vzarytovskii this is the api you would implement and export.  LMK if the contract of it makes sense to you and is feasible for F# to implement.